app/src/main/res/values-en-rGB/strings.xml

```diff
@@ -46,6 +46,7 @@
     <string name="msg_external_path">This feature will only work if you have properly set up your library on the server with network paths or path substitution and the client you are using can directly access these locations over the network.</string>
     <string name="lbl_zoom">Zoom</string>
     <string name="desc_bitstream_ac3">Requires capable hardware</string>
+    <string name="desc_audio_night_mode">Levels out audio volume automatically</string>
```

Comment:
Translations should be made at https://translate.jellyfin.org/, to avoid merge conflicts with the bot please remove the changes to this file in the git history.

Reply:
Fixed, thanks!